modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/internal/configuration/LayoutSEOGeneralGroupConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.layout.seo.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Eudaldo Alonso
+ */
+@ExtendedObjectClassDefinition(
+	generateUI = false, scope = ExtendedObjectClassDefinition.Scope.GROUP
+)
+@Meta.OCD(
+	id = "com.liferay.layout.seo.internal.configuration.LayoutSEOGeneralGroupConfiguration"
+)
+public interface LayoutSEOGeneralGroupConfiguration {
```

Comment:
Why create this new configuration class instead of using the already existing `LayoutSEOGroupConfiguration`?

Reply:
Because we want to have it hide for end users, and activate it only for this specific client. In the future, the idea is create a new UI and reorganize the SEO configuration.

Reply:
Sounds good :+1: